tmk_core/common/chibios/pin_defs.h

```diff
@@ -53,6 +53,183 @@
 #        define D5 PAL_LINE(GPIOC, 13)
 #        define B0 PAL_LINE(GPIOC, 14)
 #    endif
+#elif defined(MK66F18)
```

Comment:
This whole block is not required, and the extra values > 15 should just be added to the non ctpc branch

Reply:
Done.